app/plans/get.controller.test.js

```diff
@@ -25,28 +25,59 @@ describe('getSentencePlanSummary', () => {
 
   describe('set up rendering', () => {
     it('should set the correct render values when there is an active plan', async () => {
-      req.params.id = 1
+      req.params.id = 11033
       getSentencePlanSummary.mockReturnValueOnce(activePlan)
-      const expected = { activePlan: true, individualId: 1 }
+      const expected = {
+        currentPlan: {
+          completedDate: '',
+          createdDate: '2019-02-27',
+          isDraft: false,
+          legacy: false,
+          planId: '2345',
+        },
+        individualId: 11033,
+        planType: 'active',
+      }
       await sentencePlanSummary(req, res)
       expect(res.render).toHaveBeenCalledWith(`${__dirname}/index`, expected)
     })
-    it('should set the correct render values when there is no active plan', async () => {
+    it('should set the correct render values when there is a draft plan', async () => {
+      req.params.id = 11035
+      getSentencePlanSummary.mockReturnValueOnce(draftPlan)
+      const expected = {
+        currentPlan: {
+          createdDate: '2019-02-27',
+          isDraft: true,
+          legacy: false,
+          planId: '2345',
+        },
+        individualId: 11035,
+        planType: 'draft',
+      }
+      await sentencePlanSummary(req, res)
+      expect(res.render).toHaveBeenCalledWith(`${__dirname}/index`, expected)
+    })
+    it('should set the correct render values when there is no active or draft plan', async () => {
       req.params.id = 11034
       getSentencePlanSummary.mockReturnValueOnce(noActivePlan)
-      const expected = { activePlan: false, individualId: 11034 }
+      const expected = {
+        currentPlan: {},
+        individualId: 11034,
+        planType: 'none',
+      }
       await sentencePlanSummary(req, res)
       expect(res.render).toHaveBeenCalledWith(`${__dirname}/index`, expected)
     })
-  })
-  describe('check if active plan is present', () => {
-    it('should return true when a plan has null or no completed date', () => {
-      expect(hasActivePlan(activePlan)).toEqual(true)
-      expect(hasActivePlan(completedDateAbsent)).toEqual(true)
-    })
-    it('should return false when all plans are completed', () => {
-      expect(hasActivePlan(noActivePlan)).toEqual(false)
-      expect(hasActivePlan(emptyObject)).toEqual(false)
+    it('should set the correct render values when there is no plan information', async () => {
+      req.params.id = 11032
+      getSentencePlanSummary.mockReturnValueOnce(emptyObject)
+      const expected = {
+        currentPlan: {},
+        individualId: 11032,
+        planType: 'none',
+      }
+      await sentencePlanSummary(req, res)
+      expect(res.render).toHaveBeenCalledWith(`${__dirname}/index`, expected)
```

Comment:
Error scenario test? What should happen if getSentencePlanSummary throws an error?